pyproject.toml

```diff
@@ -166,6 +166,7 @@ testing = [
     "scikit-learn",
     "yamllint==1.32.0",
     "coverage[toml]==7.2.6",
+    "fastapi",
```

Comment:
We can use https://github.com/bentoml/BentoML/blob/main/src/bentoml/_internal/types.py#L83 to avoid hard depends on fastapi

Reply:
that's a testing only dependency. to test fastapi mount behavior

Reply:
I think test is fine for this.